cn/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md

```diff
@@ -0,0 +1,210 @@
+---
+title: 通过文件将Pod信息呈现给容器
+---
+
+{% capture overview %}
+
+此页面描述如何使用DownwardAPIVolumeFile把自己的信息呈现给pod中运行的容器。DownwardAPIVolumeFile可以呈现pod的字段和容器字段。
+
+{% endcapture %}
+
+
+{% capture prerequisites %}
+
+{% include task-tutorial-prereqs.md %}
+
+{% endcapture %}
+
+{% capture steps %}
+
+## Downward API
+
+有两种方式可以将Pod和Container字段呈现给运行中的容器：
+
+* [环境变量](/docs/tasks/configure-pod-container/environment-variable-expose-pod-information/)
+* DownwardAPIVolumeFiles
+
+这两种呈现Pod和Container字段的方式都称为*Downward API*。
+
+## 存储Pod字段
+
+在这个练习中，你将创建一个包含一个容器的pod。这是该pod的配置文件：
+
+{% include code.html language="yaml" file="dapi-volume.yaml" ghlink="/docs/tasks/inject-data-application/dapi-volume.yaml" %}
```

Comment:
copy a dapi-volume-resources.yaml to /cn/docs/tasks/inject-data-application/dapi-volume-resources.yaml
and modify the path:/docs/tasks/inject-data-application/ to /cn/docs/tasks/inject-data-application/

Reply:
lack of  dapi-volume-resources.yaml results in netlify/deploy failed.